docs/arguments.md

```diff
@@ -92,6 +92,38 @@ The ingester query API was improved over time, but defaults to the old behaviour
 - `-distributor.extra-query-delay`
    This is used by a component with an embedded distributor (Querier and Ruler) to control how long to wait until sending more than the minimum amount of queries needed for a successful response.
 
+- `distributor.accept-ha-samples`
```

Comment:
How is this a per-user flag, it's global for all user no? Can we rename it to be something more descriptive?

Reply:
It's defined in limits, so you can set an override per user. Is the description not accurate? 

We can rename to anything, any suggestions as to what would be more descriptive?

Reply:
I think calling it "per user" is very misleading. We can call it "The default value to enable handling of samples with external labels for identifying replicas in an HA Prometheus setup. This defaults to false, and is technically defined in the Distributor limits"

Also, list it in the limits section like `distributor.ingestion-rate-limit`